var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -220,3 +220,20 @@ def check_install(self):
             source_file, [include_flags], link_flags.split()
         )
         compare_output_file(output, blessed_file)
+
+    @property
+    def blas_libs(self):
+        shared = '+shared' in self.spec
+        prefix = self.prefix
+        search_paths = [[prefix.lib, False], [prefix.lib64, False],
+                        [prefix, True]]
+        for path,recursive in search_paths:
+            libs = find_libraries('libopenblas', root=path, shared=shared,
+                                  recursive=recursive)
+            if libs:
+                return libs
+        return None  # Raise error
```

Comment:
It may be useful to implement the default `.libs` and `.headers` handlers to search first `prefix.lib`, `prefix.lib64`, and `prefix.include` before resorting to the recursive search in `prefix`.

Reply:
Also, for external packages, the default behaviour of the '.headers' handler is quite inappropriate, e.g. if the `prefix` of the external package is `/usr`. Reducing the search to `/usr/include` (as I proposed above) is not much better, in this case.